### DISCUSSION TITLE
Flex "Gap"

### DISCUSSION THREAD

**Discussion by Snotax** [GeekyAnts/NativeBase#5479] [Announcements]
### Description

The "Flex" Component doesn't have a "gap" property exposed. But if I set it nontheless, it would work. 
Is this not exposed by accident or is it somehow not intended to work ?

### Problem Statement

<Flex> doesnt expose "gap"

<Flex maxW={"100%"} flexWrap={"wrap"} direction="row" justify={"center"} gap={5}>

### Proposed Solution or API

Expose gap property on <Flex>

**Comment by alissonalvarenga** [comment]
Would be great to also add the rowGap and columnGap

**Comment by AshwiniKumar007** [comment]
Hey, @alissonalvarenga  Thanks for the feedback. 
`gap` prop works only on the web and that is why we haven't exposed it as we are trying to make this universal. 
You can always use `VStack`, `HStack`, and Stack which have `space` prop to provide a gap.

We will plan to add `gap`, `rowGap`, and `columnGap` in the future.

**Comment by codehz** [comment]
https://reactnative.dev/blog/2023/01/12/version-071 react-native support gap natively, could you add support to it?
